### PR TITLE
Added consumer functionality for detailed community views

### PR DESCRIPTION
**Pull request by nicolelst**
detailed thread view and detailed meetup view are now consumers of thread and user manager !!!!!
functionality has not been affected from previous implementation. 
detailed meetup view also has a new display for the list of attendees.

### CODE REVIEW

File: lib/view/community/detailed_thread_view.dart [hunk 1]
```diff
@@ -1,211 +1,275 @@
...
+                        ? PopupMenuButton(
+                        onSelected: (value) {
+                          setState(() {
+                            if (value == "edit") {
+                              // TODO ROUTING: navigate to edit thread screen
+                            } else if (value == "delete") {
+                              threadMgr.removeThread(widget.threadID);
+                              // TODO ROUTING: fix this. screen turns red bc thread is delated. route to threadlist view?
+                            }
+                          });
```
Comment (nicolelst) [inline lib/view/community/detailed_thread_view.dart:152]: need some help to add routing to editThread after the user presses Edit thread, and routing back to thread list view after the user presses Delete thread

Comment (nicolelst) [reply lib/view/community/detailed_thread_view.dart:152]: @RealDyllon @celerie could you assist? thank you very much

**Comment by RealDyllon** [comment]
@nicolelst can we mark this as ready for review?

**Comment by nicolelst** [comment]
> @nicolelst can we mark this as ready for review?

@RealDyllon nope, still needs routing to be fully functional

> need some help to add routing to editThread after the user presses Edit thread, and routing back to thread list view after the user presses Delete thread

**Comment by RealDyllon** [comment]
@nicolelst OK I'll add routing

**Comment by nicolelst** [comment]
@RealDyllon thank you! after routing, it should be ready for review.

File: lib/view/community/detailed_meetup_view.dart [hunk 2]
```diff
@@ -59,301 +46,390 @@ class _DetailedMeetupViewState extends State<DetailedMeetupView> {
...
+                      meetupMgr.canEditMeetup(widget.meetupID))
+                  ? PopupMenuButton(
+                      onSelected: (value) {
+                        setState(() {
+                          if (value == "edit") {
+                            // TODO ROUTING: navigate to edit meetup screen
+                          } else if (value == "delete") {
+                            meetupMgr.cancelMeetup(widget.meetupID);
+                            // TODO ROUTING: navigate to meetup list view
+                          }
```
Comment (nicolelst) [inline lib/view/community/detailed_meetup_view.dart]: @RealDyllon could you also add the same routing from detailed thread view to detailed meetup view? thanks!

File: lib/view/community/detailed_thread_view.dart (hunk 1 above)
Comment (RealDyllon) [reply lib/view/community/detailed_thread_view.dart:152]: done 👍🏼

File: lib/view/community/detailed_meetup_view.dart (hunk 2 above)
Comment (RealDyllon) [reply lib/view/community/detailed_meetup_view.dart]: done 👍🏼

**Pull request merged**